tests/FakeItEasy.IntegrationTests.netstd/project.json

```diff
@@ -10,23 +10,15 @@
         "../FakeItEasy.IntegrationTests/**/*.cs"
       ]
     },
-    "define": [ "NETCORE" ]
-  },
-
-  "dependencies": {
-    "xunit.abstractions": "2.0.1-rc2",
-    "xunit.extensibility.core": "2.2.0-beta2-build3300",
-    "xunit.extensibility.execution": "2.2.0-beta2-build3300"

```

Comment:
I don't understand... without this, there isn't a single reference to XUnit. How can it work? I see that it builds, but I don't really like not understanding why :wink: